docs/features/blocks/layoutblocks/htmllayout.rst

```diff
@@ -0,0 +1,17 @@
+HTML Layout Block
```

Comment:
This doc is not needed... it is the exact same HTML block, just in a different place.

Reply:
deleted